src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -4860,6 +4860,78 @@ pub const TFD_CLOEXEC: ::c_int = O_CLOEXEC;
 pub const TFD_TIMER_ABSTIME: ::c_int = 0x01;
 pub const TFD_TIMER_CANCEL_ON_SET: ::c_int = 0x02;
 
+// Ethernet - START
+// - net/ethernet.h
+pub const ETHER_ADDR_LEN: ::c_int = 6;
+
+// - linux/if_ether.h
+pub const ETH_ALEN: ::c_int = ETHER_ADDR_LEN;
+// Ethernet - END
+
+cfg_if! {
+    if #[cfg(any(freebsd13, freebsd14, freebsd15))] {
+        // Netlink - START
+        /// sys/socket.h
+        pub const AF_NETLINK: ::c_int = 38;
+        pub const PF_NETLINK: ::c_int = AF_NETLINK;
+        
+        /// Not Supported (added for now just to compile)
+        pub const AF_BRIDGE: ::c_int = 7;
```

Comment:
@dfr and @emaste open question to discuss during coming Monday call:
- According to [rtnetlink(4)](https://man.freebsd.org/cgi/man.cgi?query=rtnetlink&apropos=0&sektion=0&manpath=FreeBSD+14.1-RELEASE&arch=default&format=html) we only support:
1. AF_INET
2. AF_INET6

We need to add AF_BRIDGE in FreeBSD netlink/rtnetlink support.

Cases:
- https://github.com/Project-Vacuum/nispor/blob/base/src/lib/query/inter_ifaces.rs#L90